packages/lab5/README.md

```diff
@@ -0,0 +1 @@
+# replace this
```

Comment:
Das File soll wahrscheinlich weg, oder?

Reply:
It's auto-generated code so I don't really care. Basically we just take what Projen gives us ^^